public/locales/auset-fr.json

```diff
@@ -0,0 +1,426 @@
+    },
+    "duchy": {
+      "name": "Duché souterrain",
+      "summaryTitle": "Joueur le Duché souterrain",
+      "summary": "<p>En tant que duché souterrain, vous voulez montrer aux créatures étrangères de la forêt qu'elles seraient mieux en tant que sujets. Chaque fois que vous faites basculer l'un de vos <strong>ministres</strong> à la cause de la pacification et de l'unification le terrain au-dessus, vous marquez des points. Plus le rang du ministre que vous influencez est élevé, plus vous marquez de points.</p><p>Pour influencer un ministre, vous devez révéler des cartes représentant votre soutien parmi les séparatistes des bois. Plus le rang de ministre est élevé, plus vous devez révéler de cartes. Cependant, vous ne pouvez révéler que des cartes qui correspondent à des clairières avec n'importe quel nombre de pièces du duché, montrant votre emprise dans le chaos tourbillonnant. Chaque ministre influencé vous donne une action supplémentaire à chaque tour. Certains ministres laissent même marquer des points de victoire en vantant vos investissements dans les Bois.</p><p>Vos taupes sont bien protégées dans <strong>le Terrier</strong>, une clairière que vous seul pouvez pénétrer et que vous dominez toujours. Vous pouvez vous déplacer vers n'importe quel <strong>tunnel</strong> que vous creusez à travers le bois. Une fois que vous avez pacifié les clairières et établi la règle, vous pouvez construire des <strong>citadelles</strong> et des <strong>marchés</strong> pour attirer plus de créatures à votre cause. Cependant, chaque fois que vous perdez des bâtiments, vous devez payer le <strong>prix de l'échec</strong> en perdant l'influence d'un ministre du rang le plus élevé et en défaussant une carte au hasard.</p>",
+      "setupTitle": "Configurer le Duché souterrain",
+      "setup": "<i>1.</i> Choisissez une clairière de patrie qui n'est pas adjacente aux patries ennemies.<br/><i>2.</i> Mettez 2 <strong>guerriers</strong> et 1 <strong>tunnel</strong> dans votre pays d'origine. Placez 5 <strong>guerriers</strong> parmi les clairières adjacentes aussi uniformément que possible.<br/><i>3.</i> Placez votre <strong>Panneau de Terrier< /strong> près de la carte. Remplissez vos pistes Bâtiments avec vos <strong>citadelles</strong> et <strong>marchés</strong>. Mettez vos 9 <strong>cartes de ministre</strong> sur votre pile de Ministres Insoumis. Mettez vos 9 <strong>couronnes</strong> sur les cases carrées indiquant les points de victoire."
```

Comment:
Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.